webviz_config/_docs/_build_docs.py

```diff
@@ -162,3 +162,23 @@ def build_docs(build_directory: pathlib.Path) -> None:
     (build_directory / "sidebar.md").write_text(
         template.render({"packages": plugin_documentation.keys()})
     )
+
+
+def _split_docstring(docstring: str) -> List[str]:
+    """Divide docstring by splitting on ---, also unindent
+    in case of indented docstrings (similar to this one)
+    """
+    lines = docstring.strip().split("\n")
+    if len(lines) == 1:
+        return lines
```

Comment:
Do we need this special casing? Rest of the code should run ok?
```python
a = [42]
# a[1:] == []
```

Reply:
No, we don't 👍 Good catch 🎉

Reply:
Edit, with the next suggested change it was needed a similar handling. Changed to try-except.